include/internal/catch_common.h

```diff
@@ -61,7 +62,7 @@ namespace Catch {
     std::ostream& operator << ( std::ostream& os, SourceLineInfo const& info );
 
     // This is just here to avoid compiler warnings with macro constants and boolean literals
-    bool isTrue( bool value );
+    inline bool isTrue( bool value ) { return value; }
```

Comment:
Shouldn't that also be marked `static`?

Reply:
definitely not, this is not at file scope, and using static for that purpose has been deprecated by the c++ standard.